packages/repository/src/__tests__/unit/repositories/relations-helpers/mongo-related-helpers.unit.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {ObjectID} from 'bson';
+import {belongsTo, Entity, hasMany, model, property} from '../../../..';
+import {
+  isBsonType,
+  deduplicate,
+  normalizeKey,
+  buildLookupMap,
+  reduceAsArray,
+} from '../../../../relations';
+
+describe('unit tests, simulates mongodb env for helpers of ininclusion resolver ', () => {
+  describe('helpers for formating instances', async () => {
+    it('isBsonType', async () => {
+      const id1 = new ObjectID();
+      const id2 = new ObjectID();
+      expect(isBsonType(id1)).to.be.true();
+      expect(isBsonType(id2)).to.be.true();
+    });
+    it('simple deduplcate + isBsonType', async () => {
```

Comment:
```suggestion
    it('simple deduplicate + isBsonType', async () => {
```

Also please add an empty line to visually delimit every two test cases - this comment applies to all new code.

```ts
});

it(...
```